features/steps/api_component.go

```diff
@@ -2,11 +2,12 @@ package steps
 
 import (
 	"context"
-	"github.com/ONSdigital/dp-interactives-api/models"
-	uuid "github.com/satori/go.uuid"
 	"net/http"
 	"strings"
 
+	"github.com/ONSdigital/dp-interactives-api/models"
+	uuid "github.com/satori/go.uuid"
```

Comment:
we should prob use https://github.com/google/uuid - i think the satori one is no longer maintained - didnt spot before 🙈

Reply:
cool .. later ? running out of time :)